api/TicketMasterAPI.js

```diff
@@ -0,0 +1,412 @@
+
```

Comment:
This file has some separation of concerns issues - namely, the entire DB-server interaction is coded in this file - this makes it more than 400 lines, which is a little unwieldy when trying to troubleshoot issues or handle scaling.  We could solve this issue by taking the knex operations into a separate file that then exported methods (we could call them things like "findTicketById", "get all )of its own that had closure over the knex variable - that way the DB interactions could be encapsulated in a separate location and we could get this file down to a shorter length.